synthtool/gcp/templates/node_library/.kokoro/release/publish.cfg

```diff
@@ -47,13 +47,9 @@ before_action {
   }
 }
 
-before_action {
-  fetch_keystore {
-    keystore_resource {
-      keystore_config_id: 73713
-      keyname: "{{ publish_token }}"
-    }
-  }
+env_vars: {
+  key: "SECRET_MANAGER_KEYS"
+  value: "{{ publish_token }}"
```

Comment:
Do you still need the braces?

I mean, `publish_token` instead of `{{ publish_token }}`?

Reply:
This is hold over from us having a publication token per-repository (which is part of why keystore was hard to use for us).

This variable expands to a different name on a per-repo basis, e.g., `googleapis-publish-token`, `kms-publish-token`, `automl-publish-token`. I kept this behavior for now.

Reply:
I went ahead and moved us to a single token (which has 2FA enabled, so continues to be secure).

This simplifies the migration.